lib/sles4sap_publiccloud.pm

```diff
@@ -272,7 +273,7 @@ sub check_takeover {
     my ($self) = @_;
     my $hostname = $self->{my_instance}->{instance_id};
     my $retry_count = 0;
-    my $fenced_hana_status = $self->is_hana_online();
+    my $fenced_hana_status = $self->is_hana_database_offline();
     die("Fenced database '$hostname' is not offline") if ($fenced_hana_status == 1);
```

Comment:
This confuses me. Before this was checking for "HANA Online" and now for "HANA Database Offline", but the conditional in the `if` does not change?